src/app/components/team/Statuses/DeleteStatusDialog.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import FormControl from '@material-ui/core/FormControl';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import ConfirmProceedDialog from '../../layout/ConfirmProceedDialog';
+
+const DeleteStatusDialog = ({
+  deleteStatus,
+  onCancel,
+  onProceed,
+  open,
+  statuses,
+}) => {
+  const [moveToStatus, setMoveToStatus] = React.useState(0);
```

Comment:
Since status is not an integer, can we have a default value of `null` instead of zero?